tests/integration/components/one-way-checkbox-test.js

```diff
@@ -49,3 +49,8 @@ test('It can accept an outside toggle of checked', function(assert) {
 
   assert.strictEqual(this.get('checked'), true);
 });
+
+test('I can add a class attribute', function(assert) {
+  this.render(hbs`{{one-way-checkbox class="testing"}}`);
+  assert.equal(true, this.$('input').attr('class').indexOf('testing') > -1);
```

Comment:
jQuery has a `hassClass` function, so you could do `this.$('input').hasClass('testing')` instead.